web/src/app/settings/single-setting/single-setting.component.html

```diff
@@ -0,0 +1,22 @@
+<a>
+    <div [ngClass]="{'forcibly-enabled-text': this.setting.forciblyEnabled}">
+        {{setting.name}}
+    </div>
+    <div *ngIf="isControlButtonsDisplayed()"
+         class="controls">
+        <div class="forcibly-enabled-button"
+                [ngClass]="{'forced-enabled-button-on' : !isEnabledForcibly(), 'forced-enabled-button-off' : isEnabledForcibly()}"
+             (click)="onEnableForciblyClicked($event)">
```

Comment:
Smth going wrong with formatting. Maybe it will be more understandable if use `[class.forced-enabled-button-on]` and `[class.forced-enabled-button-off]` directives